### ISSUE TITLE
Wrong path when creating a new Vertx project

### ISSUE THREAD

**Issue by DavidRouyer** [pmlopes/VertxSnippet#2]
I'm trying to create a new Vertx project by right-clicking on the file explorer (I'm in a C:/vertx folder) and I get this error on Windows:

`ENOENT: no such file or directory, lstat 'C:\C:\vertx'`

Windows 10
Code 1.10.2

Thanks!

**Comment by pmlopes** [comment]
Fixed by https://github.com/pmlopes/VertxSnippet/commit/4b955cd845c752075afd3c5698b1f53232b07a7a

**Comment by sibbl** [comment]
I'm using the currently latest published version 0.0.8 and still run into this issue:

```
ENOENT: no such file or directory, lstat 'C:\c:\projects\blabla'
EPERM: operation not permitted, mkdir 'C:\Program Files\Microsoft VS Code\undefined'
```
